### ISSUE TITLE
Warnings shown only once

### ISSUE THREAD

**Issue by anttih** [purescript/purescript#1588]
Warnings are shown only once even for modules that are part of the project and are not dependencies. Say you have two warnings, both in their own module. When you go and fix the other one and compile again,  both of those warnings disappear. You have to go and `touch` the other file to make the warning appear again. This behavior is fine for dependencies but not for the project src files.

**Comment by phadej** [comment]
AFAIK if file is compiled, it's not touched if there are no need to touch it. To get warnings, you need to recompile, which is expensive. This how all compilers work: gcc, clang, ghc, psc...

OTOH `psc` could have `-Werror` to turn warnings into errors (i.e. prevent successful compilation with warnings), so you can hunt all warnings if you are into that.

**Comment by anttih** [comment]
Yep. Basically this is a duplicate of #1434. I was thinking along the lines of a glob pattern suggested by @hdgarrood. Closing this one.
